internal/ingress/controller/template/template.go

```diff
@@ -1079,6 +1080,11 @@ func getIngressInformation(i, h, p interface{}) *ingressInformation {
 		return &ingressInformation{}
 	}
 
+	ingressPathType, ok := t.(*v1.PathType)
+	if !ok {
+		klog.Errorf("expected a '*v1.PathType' type but %T was returned", t)
+	}
```

Comment:
I'm unsure if this is the best way to handle the type assertion.
It seems like we may get the correct type, or nothing at all (empty string I think).
Happy to change this if anyone has a suggestion.